src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -600,7 +601,7 @@ public double getYaw() {
    *     Example</a>
    */
   public Command followTrajectoryCommand(PathPlannerTrajectory traj, boolean isFirstPath) {
-    field.getObject("path").setTrajectory(traj);
+    // field.getObject("path").setTrajectory(traj);
```

Comment:
In simulation, this code hides the actual robot location.  _should disable only when in simulation.....or make it transparent_